uo_sys_framework/ACREModule/cfgFunctions/common/fn_addRack.sqf

```diff
@@ -0,0 +1,148 @@
+/*	Description: Adds racks to objects as specified.
+ *	Arguments:
+ * 0: vehicle or object <OBJECT> (default: objNull)
+ *	Return Value:
+ * 		NULL
+ *	Author
+ *		TrainDoctor
+*/
+
+#include "\x\UO_FW\addons\main\script_macros.hpp"
+
+if (!UO_FW_ACRE_SETTINGS_ENABLED) exitwith {};
+	
+if (!isServer) exitWith {
```

Comment:
`// In MP scenario, the expression is called only on server.`

If you want an only Server function you could use the `UO_FW_EXEC_CHECK(SERVER)` macro will will account for Singleplayer, whether FW is enabled, isServer, etc...

You should also include our CBA server setting checks for any PostInit/attribute expression called fncs

`if !(UO_FW_SERVER_ACREMODULE_ALLOWED) exitwith {};`

The server settings can be found in <https://github.com/unitedoperations/UnitedOperationsFramework/blob/master/uo_sys_framework/XEH_preInit.sqf>